index.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import dotenv from 'dotenv';
+import chalk from 'chalk';
+import swaggerDoc from 'swagger-ui-express';
+import { swaggerControllers } from './controllers/index.js';
+
+dotenv.config();
+
+const app = express();
+
+const { PORT = 3000 } = process.env;
+
+app.use(morgan('short'));
+
+app.use(cors());
+
+app.use(express.json());
+
+app.use('/docs', swaggerDoc.serve, swaggerDoc.setup(swaggerControllers));
+console.log(
+  chalk.bgYellow.italic(`Swagger here ----> http://localhost:${PORT}/docs`)
+);
+
+app.use((_req, res) => {
```

Comment:
why it's `_req`, because we don't use the parameter?

Reply:
Exactly, it is. Because of the eslint rules.